packages/react/src/themes/teams/components/Menu/menuVariables.ts

```diff
@@ -1,89 +1,89 @@
-  primaryUnderlinedBorderColor: string
-
-  disabledColor: string
+  colorDisabled: string
   lineHeightBase: string
   horizontalPadding: string
 
   verticalBackgroundColor: string
   verticalItemPadding: string
   verticalBoxShadow: string
   verticalDividerMargin: string
-  verticalItemBorder: string
+  verticalItemBorderWidth: string
+  verticalItemBorderColor: string
+  verticalPointingBorderColor: string
+  verticalBackgroundColorFocus: string
 
   pointingIndicatorBackgroundColor: string
 
   underlinedBottomBorderWidth: string
+  primaryBorderColor: string
 
   dividerHeight: string
   borderWidth: string
 }
 
 export default (siteVars: any): MenuVariables => {
   return {
-    color: siteVars.gray02,
-    borderColor: siteVars.gray08,
-    verticalBorderColor: siteVars.gray08,
-
-    focusedBorder: `solid ${pxToRem(1)} ${siteVars.colors.white}`,
-    focusedOutline: `solid ${pxToRem(1)} ${siteVars.colors.black}`,
-    focusedBackgroundColor: siteVars.gray09,
+    colorScheme: siteVars.colorScheme,
+    color: siteVars.colors.grey[500],
+    colorActive: siteVars.colors.black,
+    colorFocus: siteVars.colors.white,
+    colorDisabled: siteVars.colors.grey[250],
 
-    hoverBackgroundColor: siteVars.gray14,
+    borderColor: undefined,
+    borderColorHover: undefined,
+    borderColorActive: undefined,
+    borderColorFocus: siteVars.colors.white,
 
-    activeColor: siteVars.colors.black,
-    activeBackgroundColor: siteVars.gray10,
-    iconOnlyActiveColor: siteVars.colors.primary[500],
+    outlineColorFocus: siteVars.colors.black,
 
-    primaryActiveColor: siteVars.colors.white,
-    primaryActiveBackgroundColor: siteVars.brand08,
-    primaryActiveBorderColor: siteVars.colors.primary[500],
+    backgroundColor: undefined,
+    backgroundColorFocus: undefined,
+    backgroundColorHover: undefined,
+    backgroundColorActive: undefined,
 
-    primaryFocusedColor: siteVars.colors.white,
-    primaryFocusedBackgroundColor: siteVars.colors.primary[200],
+    iconOnlyColorActive: siteVars.colors.primary[600],
 
-    primaryBorderColor: siteVars.gray10,
-    primaryHoverBorderColor: siteVars.gray08,
-    primaryUnderlinedBorderColor: siteVars.gray08,
+    underlinedBorderColor: siteVars.colors.grey[200],
 
-    disabledColor: siteVars.gray06,
     lineHeightBase: siteVars.lineHeightMedium,
     horizontalPadding: `${pxToRem(14)} ${pxToRem(18)} ${pxToRem(14)} ${pxToRem(18)}`,
 
-    verticalBackgroundColor: siteVars.colors.white,
+    verticalBackgroundColor: undefined,
     verticalItemPadding: `${pxToRem(9)} ${pxToRem(16)} ${pxToRem(9)} ${pxToRem(16)}`,
     verticalBoxShadow: siteVars.shadowLevel3,
     verticalDividerMargin: `${pxToRem(8)} 0`,
-    verticalItemBorder: `solid ${pxToRem(2)} transparent`,
+    verticalItemBorderWidth: pxToRem(2),
+    verticalItemBorderColor: 'transparent',
+    verticalPointingBorderColor: siteVars.colorScheme.primary.borderActive, // TODO: why is the default having primary color?!
```

Comment:
> TODO: why is the default having primary color?!

most likely implementation mistake. probably the author didn't really care about default menus as most of them are used with `primary` prop set

Reply:
I would not necessarily say mistake, if in teams theme the vertical pointing menu should always look like this. I added the comment as a red flag, is it may not be the intended behavior. If the combination of these two props always results in the same border color, then this variable is enough, and the variable will always win...